modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/resource/test/HeadlessBuilderResourceTest.java

```diff
@@ -3123,6 +3067,194 @@ private void _relateObjectEntries(
 			objectRelationship, TestPropsValues.getUserId());
 	}
 
+	private void _testGetInDifferentCompany(
+			String apiSchemaExternalReferenceCode,
+			ObjectDefinition objectDefinition, long userId)
```

Comment:
You are passing the userId by argument, however you are using the credentials "test@able.com", I think you could get the information you need inside the method then

Reply:
The userid is used for a service call, so we need it unless we migrate the service call to a post call.

Reply:
Yes, but does it make sense to trust in the user that you are receiving but argument, as inside the method you know the user has to be the test@able.com ??